internal/command/checks/command.go

```diff
@@ -0,0 +1,106 @@
+package checks
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.stackrox.io/kube-linter/internal/builtinchecks"
+	"golang.stackrox.io/kube-linter/internal/check"
+	"golang.stackrox.io/kube-linter/internal/command/common"
+	"golang.stackrox.io/kube-linter/internal/defaultchecks"
+)
+
+var (
+	dashes = func() string {
+		var sb strings.Builder
+		for i := 0; i < 30; i++ {
+			sb.WriteRune('-')
+		}
+		return sb.String()
+	}()
+
+	formatsToRenderFuncs = map[string]func([]check.Check, io.Writer) error{
+		common.PlainFormat:    renderPlain,
+		common.MarkdownFormat: renderMarkdown,
+	}
+)
+
+func renderPlain(checks []check.Check, out io.Writer) error { //nolint:unparam // The function signature is required to match formatToRenderFuncs
+	for i, chk := range checks {
+		fmt.Fprintf(out, "Name: %s\nDescription: %s\nTemplate: %s\nParameters: %v\nEnabled by default: %v\n",
+			chk.Name, chk.Description, chk.Template, chk.Params, defaultchecks.List.Contains(chk.Name))
+		if i != len(checks)-1 {
+			fmt.Fprintf(out, "\n%s\n\n", dashes)
+		}
+	}
+	return nil
+}
+
+const (
+	markDownTemplateStr = `The following table enumerates built-in checks:
```

Comment:
Ok so what I actually had in mind was something like `--format go-template="{{ template str }}"` or `--format go-template=@file`. It is fairly established in the Go world to allow users to specify Go templates for all "print"-like commands (e.g., `go list`, `kubectl get`.. etc) (although the file variant is something non-standard, but useful here).
This works for the initial iteration but having to resort to `{{backtick}}` is.. ugly..

Reply:
Ahh got it. Yeah, I planned to support Go template output for the `lint` command but wasn't sure if it was worth it for these commands... Will do in a follow-up.